plugins/ci/skills/e2e-test-lifecycle/SKILL.md

```
@@ -0,0 +1,1087 @@
+- [ ] Test has appropriate `[FeatureGate:XYZ]` or `[Feature:XYZ]` annotations
+- [ ] Test duration is under 5 minutes (longer tests need architect approval)
+- [ ] Parallel tests are non-disruptive and can run alongside any other test
+- [ ] Serial tests restore cluster to original state after completion
+- [ ] Test passes at >= 99% for blocking status (or has `Informing()` lifecycle)
+
+## Recommended Payload Jobs for Validation
+
+After changing suite membership, run payload jobs to validate:
+
+For parallel suite changes:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)